lib/algolia/getRecords.js

```diff
@@ -2,17 +2,19 @@ const fs = require('fs');
 const path = require('path');
 const url = require('url');
 const algoliasearch = require('algoliasearch/lite');
+
 const {
     cacheContentAlias,
     getRecordObject,
     pushRecursively,
-    createRecords
+    createRecords,
+    getSectionContent
 } = require('./algoliaUtils');
 
 async function updateIndex() {
-    const appId = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID
-    const adminKey = process.env.NEXT_PUBLIC_ALGOLIA_ADMIN_API_KEY
-    const algoliaIndex = process.env.NEXT_PUBLIC_ALGOLIA_INDEX
+    const appId = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID;

```

Comment:
did we create config change ticket for this?

Reply:
No, it was added to Vercel directly

Reply:
Created a ticket